eq-author/src/App/publish/PublishPage.js

```diff
@@ -113,10 +113,21 @@ const PublishPage = ({ match, history }) => {
   };
 
   const publishStatus = questionnaire && questionnaire.publishStatus;
+
+  const canPublish = () => {
+    if (!questionnaire) {
+      return true;
+    }
+    return (
+      me.admin ||
+      questionnaire.createdBy.id === me.id ||
+      questionnaire.editors.some(el => el.id === me.id)
+    );
```

Comment:
Think we can just check to see if `questionnaire.permission === "Write"` here. Should make it easier to keep the permission logic consistent.